ray-operator/apis/ray/v1/rayservice_types.go

```diff
@@ -66,8 +71,8 @@ type RayServiceSpec struct {
 	DeploymentUnhealthySecondThreshold *int32 `json:"deploymentUnhealthySecondThreshold,omitempty"`
 	// ServeService is the Kubernetes service for head node and worker nodes who have healthy http proxy to serve traffics.
 	ServeService *corev1.Service `json:"serveService,omitempty"`
-	// UpgradeStrategy represents the strategy used when upgrading the RayService. Currently supports `NewCluster` and `None`
-	UpgradeStrategy *RayServiceUpgradeStrategy `json:"upgradeStrategy,omitempty"`
+	// UpgradeSpec defines the scaling policy used when upgrading the RayService.
+	UpgradeSpec *RayServiceUpgradeSpec `json:"upgradeSpec,omitempty"`
```

Comment:
`spec.upgradeStrategy.type` instead of `spec.upgradeSpec.type`?

Reply:
Sure that sounds more clear, `RayServiceUpgradeStrategy` was already in use to define the string constant, but I went ahead and changed that field to `RayServiceUpgradeType` instead. 97816e8